FAQ.md

```diff
@@ -176,92 +176,93 @@ Post quality is judged by the amount of effort you put into the creation of your
 
-### Q: How to check how many pearls I have? Who is the wealthiest user?
-**A**: [Clicking here](https://blackpearl.biz/pearls/) you can check how many pearls do you have, who the wealthiest user is, and all the pearls you won/spent.
+### Q: How do I check how many pearls I have? Who is the wealthiest user?
+**A**: [Clicking here](https://blackpearl.biz/pearls/) you can check how many pearls you have, who the wealthiest user is, and all the pearls you have won or spent.
 
 ### Q: I just created a thread/post but can't see it posted.
-**A**: If you are a registered user or lower anything you post will be waiting for approval from a Moderator. Once approved, the post will become available on the forum.
+**A**: If you are a Registered user or lower anything you post will be waiting for approval from a Moderator. Once approved, the post will become available on the forum.
 
 ### Q: How long does it take to have a post approved on the forum?
-**A**: There isn't a specific time range for a post to be approved. Posts will be approved once a Moderator is available and goes through the posts needing approval. Usually, posts don't take more than 24 hours to be approved.
+**A**: There isn't a specific time range for a post to be approved. Posts will be approved once a Moderator is available and goes through the posts that need approval. Usually, posts will not take more than 24 hours to be approved.
 
 ### Q: How does one add a picture to a post? Also, what image hosters I can use?
-**A**: You can do this by clicking on the [little image icon](https://uploaddeimagens.com.br/images/002/359/857/original/2019-09-19_181337.png?1568927759) at the post tollbar or using the hotkey `Ctrl+P`. If you are using [bbcode](https://www.phpbb.com/), you have to use `[IMG]` tag. Example: `[IMG]https://hoster.com/image.png[/IMG]`.
+**A**: You can do this by clicking on the [little image icon](https://uploaddeimagens.com.br/images/002/359/857/original/2019-09-19_181337.png?1568927759) at the post toolbar or using the hotkey `Ctrl+P`. If you are using [bbcode](https://www.phpbb.com/), you have to use the `[IMG]` tag. Example: `[IMG]https://hoster.com/image.png[/IMG]`.
 
-For that, you need to use image hoster. We recommend [ImgBB](https://imgbb.com/), [imgur](https://imgur.com) or [PostImages](https://postimages.org/) but you can use any other since it allows you to reach the original image link. Example: `https://hoster.com/image.png`.
+For that you need to use an image hoster. We recommend [ImgBB](https://imgbb.com/), [imgur](https://imgur.com) or [PostImages](https://postimages.org/) but you can use any other, since it allows you to reach the original image link. Example: `https://hoster.com/image.png`.
```

Comment:
I think it is ''as long as it allows'' instead of ''since it allows''. Correct me if I'm wrong though.

Reply:
I believe both are technically correct but "as long as it allows" seems to flow better so I'll change that part.